unstract/core/src/unstract/core/pubsub_helper.py

```diff
@@ -4,16 +4,11 @@
 from datetime import datetime, timezone
 from typing import Any, Optional
 
-import redis
+from django_redis import get_redis_connection
```

Comment:
Can't do this. This pubsub_helper is core library package that we are using in worker. backend,etc.. Some of the services are not in django, and they don;t have the context of redis

Reply:
Got it :+1: